test/gpu/native/gpuWritelnAndAssertOnGpu.good

```diff
@@ -5,4 +5,4 @@ hello from the GPU again!
 hello from function called by GPU!
 hello again from  function called by GPU!
 Before next loop
-gpuWritelnAndAssertOnGpu.chpl:26: error: assertOnGpu() failed
```

Comment:
This looks like a possible place for a prediff so we can stop having to adjust line numbers if that ever changes again.

Reply:
This is a case where we want to lock down the location of the error.; we want it to refer to that specific assert at that specific location. It's pointing to a place in the test itself and not something like internal module code that's likely to shift.

In a different project I've worked on I've seen something like this where you would put a comment labeling the line number:

```
foreach i in 0..10 {
  assertOnGpu();   // #ASSERT1
}
```

Which then you could lock down in your error message:

```
gpuWritelnAndAssertOnGpu.chpl:#ASSERT1: error: assertOnGpu() failed
```

So if you changed the test and shifted code you wouldn't have to update the `.good` file.

But if we want to do something like that I'd rather it be done through a more general mechanism that we used everywhere rather than the `.prediff` for one specific test.

Reply:
Ah, I didn't catch the first assert. I'm good with using a line number here 👍